app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -1,17 +1,29 @@
 module Api
   module V1
     class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_api_v1_user!, only: %i[new create_without_auth]
```

Comment:
You can leave the create unauthenticated so you don't need to authenticate through jQuery